src/viam/sdk/services/motion/server.cpp

```diff
@@ -87,10 +82,10 @@ ::grpc::Status MotionServer::MoveOnGlobe(
                 motion_configuration::from_proto(request->motion_configuration()));
         }
 
-        const std::string execution_id = motion->move_on_globe(std::move(destination),
-                                                               std::move(heading),
-                                                               std::move(component_name),
-                                                               std::move(movement_sensor_name),
+        const std::string execution_id = motion->move_on_globe(destination,
```

Comment:
Ditto, also why `&` again.

Reply:
`destination` (`struct geo_point`) is trivially copyable, as is `boost::optional<double>`. So `destination` and `heading` remain unmoved. 

`&`ness removed.